test/name_service/core.clj

```diff
@@ -1,2 +1,55 @@
 (ns name-service.core
-  (:require [clojure.test :refer :all]))
+  (:require [clojure.test :refer :all]
+            [name-service.utils :as utils]
+            [name-service.delivery.state :as uc]
+            [mount.core :as mount]))
+
+(use-fixtures :each
+  (fn
+    [f]
+    (mount/stop)
+    (mount/start-with
+      {#'name-service.delivery.state/store (utils/->TestStorage (atom #{}))})
+    (f)))
+
+(deftest test-add-key-binding
```

Comment:
This is all great for a start so I won't complain here but the next step is to randomize data with `test.check` :)